.eslintrc.json

```diff
@@ -51,6 +51,7 @@
           "groupCollapsed",
           "groupEnd",
           "info",
+          "log",
```

Comment:
Suggestion: I would like to avoid log if we can. Since the lint recommendation is to avoid it and something like `error` or `warn` or `debug` is more clear about the purpose. Can you remove this and change the one's you added to `console.debug` or remove them if applicable?

Reply:
don't worry -- this is just for debugging in gh actions, since i can't replicate the issue locally. will remove once the PR is ready (again) :)